doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -67,12 +96,22 @@
 > * Number of interfaces and IP addresses to be supported by a pod
 > * IPv6 single and dual stack needs
 
+> Refer to software profile features [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#523-virtual-networking) and hardware profile features [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#543-network-resources).
+
 <a name="3.2.3"></a>
 ### 3.2.3 Container Storage Services
 
-> This shapter should discuss storage services provided by the reference architecture. 
+Containers, when running, will require ephemeral storage on which to run themselves (i.e. storage on which the unpacked container image is stored and executed from). This ephemeral storage lives and dies with the container and is a directory on the worker node on which the container is running.  Note, this means that the ephemeral storage is mounted locally in the worker node filesystem. The filesystem can also be physically external to the worker node (e.g. iSCSI, NFS, FC) but the container will still reference it as part of the local filesystem.
+
+Additional ephemeral storage can also be attached to a container through the use of Kubernetes Volumes - this can be storage from the worker node filesystem, or it can be external storage that is accessed through the use of a Volume Plugin. Volume Plugins allow the use of a storage protocol (e.g. iSCSI, NFS) or management API (e.g. Cinder) for the attaching and mounting of storage into a Pod. This additional ephemeral storage, that is attached to a container using a Kubernetes Volume, does not live and die with the container but instead follows the lifecycle of the Pod that the container is a part of. This means the ephemeral Volume persists across container restarts, whilst the Pod is still running.
```

Comment:
this chapter actually describes Persistent Storage, not ephemeral
also direct host mounts are generally frowned upon, and can be easily forbidden via PodSecurityPolicies. IMO this should be highlighted, and the option not glorified

Reply:
Agree re. highlighting about host mounts.
Re. Persistent Volumes that is the next paragraph - both these paragraphs highlighted are about storage that lives and dies with the Pod, i.e.. not Persistent Volumes.

Reply:
Have suggested using hostPath is not recommended.

Reply:
data stored on volume mounts is not -necessarily- lost even when Pod dies, hence my objection against using the word "ephemeral" in this chapter
a volume mount _can_ be backed by the "emptyDir" driver, in which case the data is stored in memory, true. But it is just one option.
The data does persist on a direct host volume mount.

Reply:
Ah ok yes I see what you mean.  OK, I'll try and call out that this storage is not necessarily ephemeral in the true sense, but nor is it a PersistentVolume.

Reply:
I have attempted to clarify in [this](https://github.com/cntt-n/CNTT/pull/677/commits/690df12b8befbbc5f8a0c1e2a6601e6089bc5e34) commit.